com.unity.render-pipelines.core/Editor/Lighting/SRPLensFlareElementEditor.cs

```diff
@@ -0,0 +1,490 @@
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+
+namespace UnityEditor.Rendering
+{
+    /// <summary>
+    /// SRPLensFlareElementEditor shows how each element in the SRP Lens Flare Asset are show in the UI
+    /// </summary>
+    [CustomPropertyDrawer(typeof(SRPLensFlareDataElement))]
+    public class SRPLensFlareElementEditor : PropertyDrawer
+    {
+        private float m_LastOffset = 0.0f;
```

Comment:
Note: private is the default visibility for fields. You can drop them and still have same visibility